ais-infra-alerting/template.yaml

```diff
@@ -42,7 +42,7 @@ Parameters:
     Description: >
       External API subscription endpoint i.e. PagerDuty events integration API
     Type: String
-    Default: "none"
+    Default: "https://events.pagerduty.com/integration/96e06442f974460fd0dd0b45a26c8f52/enqueue"
```

Comment:
Should be default by none else you're integrating in all environments

Reply:
So either a manual step to update parameter in Prod or remove this parameter, set a condition on parameter references to point to non unless `isProd` then can point to actual PagerDuty endpoint.

Reply:
I'd go with second option, removes a manual step

Reply:
Cool gotcha thanks 👍

Reply:
Actually, let's confirm we're fine to provide the PagerDuty endpoint in our code.

Reply:
Else will have to keep as-is and provide it manually in the existing CFN parameter

Reply:
Spoke to someone from Security Team called Matthew Lea and he said its okay to add the URL in the repos as its _not reachable from the internet_, so I guess its okay to keep in.